code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -6302,16 +6302,23 @@
 	dupeable = FALSE
 
 /datum/reagent/ethanol/scientists_serendipity/handle_special_behavior(var/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/D)
-	if(volume<10)
+	if(volume < 10)
 		glass_icon_state = "scientists_surprise"
 		glass_name = "\improper Scientist's Surprise"
 		glass_desc = "There is as yet insufficient data for a meaningful answer."
-	else
+
```

Comment:
needs to blank origin tech when it goes back down so that you can't just fill it up to 50 once and then empty it out to 1u

Reply:
It's how it was coded originally, the surprise had no origin tech, while the surprise had. Should I add this?
>D.origin_tech = ""

Reply:
That will do it, yeah. This is a good chance to fix the original oversight.

Reply:
Fixed.